### PR TITLE
Site Performance : Careers page Image optimization

### LINKED ISSUE #2172: [Site Speed] Currently a 40% rating
**Current Behavior**
https://layer5.io isn't nearly as fast loading as it could be. See the recommendations on this report - https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Flayer5.io%2F

**Desired Situation**
Take each recommendation one at-a-time, implementing suggestions to improve page speed. These individual speed enhancements can be broken into separate issues or all tracked against this same epic issue.

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by YashKamboj**
Signed-off-by: YashKamboj <75125203+YashKamboj@users.noreply.github.com>

**Description**
Replaced  to for improving image optimization and site performance, and also moved images to the respected folders in order to achieve the same.
This PR updates #2172 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Nikhil-Ladha** [comment]
Prefer `updates` over `fixes` when referring to the perf issue so that it doesn't get's closed when the PR is merged.

File: src/sections/Careers/Careers-Internship-grid/index.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+import { StaticImage } from "gatsby-plugin-image";
 import { Container, Row, Col } from "../../../reusecore/Layout";
 import PageHeader from "../../../reusecore/PageHeader";
 import Button from "../../../reusecore/Button";
 import { FaArrowRight } from "react-icons/fa";
 import InternshipSectionWrapper from "./InternshipPage.style";
 import ProgramsGrid from "../Careers-Programs-grid/index";
-import UEM_Img from "../../../assets/images/partners/uem_partner.png";
-import Univ_Texas_Img from "../../../assets/images/partners/texas_partner.png";
-import Peek_Img from "../../../assets/images/layer5/layer5-peek-card-edited.svg";
```
Comment (Nikhil-Ladha) [inline src/sections/Careers/Careers-Internship-grid/index.js:11]: Add a line break after all the imports, for readabillity.

**Comment by l5io** [comment]
🚀 Preview for commit 10990646d2e9375cffa0ac7b7ef4d97e2d7f1e3b at: https://61f02ce329ac3170c7fbf789--layer5ng.netlify.app

File: src/sections/Careers/index.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
 import React from "react";
+import { StaticImage } from "gatsby-plugin-image";
 import { Container } from "../../reusecore/Layout";
 import { Row } from "../../reusecore/Layout";
 import PageHeader from "../../reusecore/PageHeader";
 import InternshipPage from "./Careers-Internship-grid";
-import Lee_and_Rigs from "../../assets/images/careers/Lee-and-Rigs-Caballero-of-Google.jpeg";
+const Lee_and_Rigs = "../../assets/images/careers/Lee-and-Rigs-Caballero-of-Google.jpeg";
```
Comment (Nikhil-Ladha) [inline src/sections/Careers/index.js:7]: Please don't do this.
Keep all the imports together and variables separate from it.
Add this after line: 10 (after a line break)

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 399fe80ab90ce9d23df47c33f0a0bda57eb70b29 at: https://61f0d1192d8d5517c167dd47--layer5ng.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**